auth-api/src/auth_api/models/user.py

```diff
@@ -42,16 +43,18 @@ class User(db.Model):
     modified = Column(DateTime)
     roles = Column('roles', String(1000))
 
+    contacts = relationship('ContactLink', back_populates='user')
+
     @classmethod
-    def find_by_keycloak_guid(cls, keycloak_guid):
+    def find_by_username(cls, username):
         """Return the first user with the provided KeyCloak GUID."""
-        return cls.query.filter_by(keycloak_guid=keycloak_guid).first()
+        return cls.query.filter_by(username=username).first()
```

Comment:
is it username or GUID ? the comment says GUID..or username is the GUID

Reply:
Yep, missed updating the docstring